rdm/pull.py

```diff
@@ -0,0 +1,56 @@
+'''
+Functions that parse the given project management service and produce
+`requirements.yml` and `problem_reports.yml` files.  The selected project
+management backend is specified in the project configuration file.
+'''
+import os
+from getpass import getpass
+
+from github import Github
+import yaml
+
+from rdm.util import get_project_settings
+
+
+def pull_requirements_and_reports(system_yml_path):
+    settings = get_project_settings(system_yml_path)
+    if settings['software_requirements_location'] == 'GitHub':
+        output_dir = os.path.dirname(system_yml_path)
+        pull_from_github(settings, output_dir)
+    else:
+        raise ValueError("Specified software requirements location not yet supported.")
+
+
+def pull_from_github(settings, output_dir):
```

Comment:
Since we are only using the `settings['repository']` value, it would be better to pass the GitHub URL directly into `pull_from_github`, instead of passing the whole `settings` dict in.